client.go

```diff
@@ -271,3 +274,12 @@ func (c *Client) DecodeSimpleResponse(resp []byte) (*SimpleResponse, error) {
 	err := json.NewDecoder(bytes.NewReader(resp)).Decode(&response)
 	return &response, err
 }
+
+// SetUserAgent sets the user agent for the client
+func (c *Client) SetUserAgent(component *Component) {
+	if component.ID == "" {
+		c.UserAgent = fmt.Sprintf("%s/%s %s", component.Name, component.Version, c.UserAgent)
```

Comment:
I think this should be : 
```suggestion
		c.UserAgent = fmt.Sprintf("%s/%s %s", component.Name, component.Version, utils.GetVersion())
```

Reply:
If we do it in this way we lost the `civogo` string